<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_about</class>
 <widget class="QDialog" name="Dialog_about">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>615</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_app">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Stock Synthesis GUI&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="textFormat">
        <enum>Qt::RichText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_icon">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="stock_synth.qrc">:/i/images/image7_no_background.png</pixmap>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_ver">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;For version 3.2x&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="textFormat">
        <enum>Qt::RichText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="plainTextEdit">
       <property name="plainText">
        <string>History

Stock Synthesis (SS), developed by Richard Methot (pictured below), uses an approach that has progressed through three major stages of development since its beginning 30 years ago: 1) development for northern anchovy, Engraulis mordax, in which the basic concept was established, 2) re-development as a generalized model focused on west coast groundfish, and 3) translating the computer code to ADMB (Fournier et al., 2011) to take advantage of the power of automatic differentiation.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="stock_synth.qrc">:/i/images/methot_award_small.jpg</pixmap>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="plainTextEdit_2">
       <property name="plainText">
        <string>Overview

Stock Synthesis provides a statistical framework for calibration of a population dynamics model using a diversity of fishery and survey data. It is designed to accommodate both age and size structure in the population and with multiple stock sub-areas. Selectivity can be cast as age specific only, size-specific in the observations only, or size-specific with the ability to capture the major effect of size-specific survivorship. 

The overall model contains subcomponents which simulate the population dynamics of the stock and fisheries, derive the expected values for the various observed data, and quantify the magnitude of difference between observed and expected data. 

Some SS features include ageing error, growth estimation, spawner-recruitment relationship, and movement between areas. SS is most flexible in its ability to utilize a wide diversity of age, size, and aggregate data from fisheries and surveys. 

The ADMB C++ software in which SS is written searches for the set of parameter values that maximize the goodness-of-fit, then calculates the variance of these parameters using inverse Hessian and MCMC methods. 

A management layer is included in the model allowing uncertainty in estimated parameters to be propagated to the management quantities, thus facilitating a description of the risk of various possible management scenarios, including forecasts of possible annual catch limits. </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_copyright">
       <property name="text">
        <string>copyright</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pushButton_manual">
         <property name="text">
          <string>User Manual</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_website">
         <property name="text">
          <string>Website</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_ok">
         <property name="text">
          <string>OK</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>pushButton_ok</tabstop>
  <tabstop>pushButton_manual</tabstop>
  <tabstop>pushButton_website</tabstop>
 </tabstops>
 <resources>
  <include location="stock_synth.qrc"/>
 </resources>
 <connections/>
</ui>
